judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/contest/clarification/ContestClarificationRoleChecker.java

```diff
@@ -63,6 +63,10 @@ public boolean canViewOwnClarifications(String userJid, Contest contest) {
                 && contestTimer.hasStarted(contest, userJid);
     }
 
+    public boolean canViewAllClarifications(String userJid, Contest contest) {
+        return isSupervisorWithClarificationPermissionOrAbove(userJid, contest);
```

Comment:
Actually, this method and the previous method is wrong... they both should return false if the contest does not have clarification module. Could you fix this as well?